src/Chapter2.hs

```diff
@@ -336,7 +337,10 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList firstPoint secondPoint array =
+    if firstPoint < 0 || secondPoint < 0
+    then []
+    else take (secondPoint - firstPoint + 1) (drop firstPoint array)
```

Comment:
Nice implementation! Though, for performance reasons, you can add one more condition when `firstPoint > secondPoint` and return an empty list in this case as well